src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -488,7 +488,7 @@
     <value>Use centralized keyboard hook</value>
   </data>
   <data name="PowerLauncher_UseCentralizedKeyboardHook.Description" xml:space="preserve">
-    <value>Try this if there are issues with the shortcut</value>
+    <value>Try this if there are issues with the shortcut; But PowerToys Run might not get focus when triggered from an elevated window</value>
```

Comment:
```suggestion
    <value>Try this if there are issues with the shortcut. (PowerToys Run might not get focus when triggered from an elevated window.)</value>
```

Reply:
What do you think about your suggestion but without the dots?

Reply:
Hmm. We can do that. For me it always looks strange if a sentence doesn't end with a dot.

Reply:
In the settings most things don't end with a dot

Reply:
```suggestion
    <value>Try this if there are issues with the shortcut (PowerToys Run might not get focus when triggered from an elevated window)</value>
```